src/DwitTech.AccountService.WebApi/Controllers/UserController.cs

```diff
@@ -1,10 +1,64 @@
-﻿using DwitTech.AccountService.Core.Dtos;
-using DwitTech.AccountService.Core.Services;
+﻿using DwitTech.AccountService.Data.Entities;
+using DwitTech.AccountService.Core.Utilities;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using DwitTech.AccountService.Core.Interfaces;
+
 
 namespace DwitTech.AccountService.WebApi.Controllers
-{
+{ 
     public class UserController : BaseController
     {
+        public static User user = new User();
+
+        private readonly IValidationService _validationService;
+        private readonly IActivationService _activationService;
+
+        public UserController(IValidationService validationService, IActivationService activationService)
+        {
+            _validationService = validationService;
+            _activationService = activationService;
+        }
+
+
+        [HttpGet("/Account/Activation/{activationCode}")]
```

Comment:
This should just be /Activation/{activationCode}. The Account in the path is inherited from the controller name.

Reply:
> This should just be /Activation/{activationCode}. The Account in the path is inherited from the controller name.

Resolved